pkg/github/token.go

```diff
@@ -218,14 +232,20 @@ func rateLimitCallBackfunc(cbContext *github_ratelimit.CallbackContext) {
 
 	// Start a goroutine that marks the given client as rate limited and sleeps for 'TotalSleepTime'
 	go func(client *GitHubClient) {
+		srlTokenMetric := metrics.TokenPoolGauge.With(prometheus.Labels{"rateLimited": "secondary", "tokenName": client.TokenName})
+
 		client.SecondaryRateLimit.mu.Lock()
 		client.SecondaryRateLimit.isLimitReached = true
+		log.Info("SRL metric incremented ")
```

Comment:
Include token name here?

Reply:
ah good catch, that was just a debug statement.  Let me remove it